### PR TITLE
Modify not to generate .jar analysis file

### PR DESCRIPTION
**Pull request by bjk7119**
## Description

- Modify not to generate .jar analysis file
- Remove dependency-check-report.json file after analysis

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/fosslight_binary/_jar_analysis.py [hunk 1]
```diff
@@ -196,4 +196,5 @@ def analyze_jar_file(path_to_find_bin):
         logger.debug(f"Error to get depency Info in jar_contets: {ex}")
         success = False
 
+    os.remove(json_file)
```
Comment (soimkim) [inline src/fosslight_binary/_jar_analysis.py:199]: Wrap it in a try except statement and delete it after checking if the file exists.

Comment (bjk7119) [reply src/fosslight_binary/_jar_analysis.py]: Added it. Thx

File: src/fosslight_binary/_jar_analysis.py [hunk 2]
```diff
@@ -196,4 +197,11 @@ def analyze_jar_file(path_to_find_bin):
         logger.debug(f"Error to get depency Info in jar_contets: {ex}")
         success = False
 
+    try:
+        if json_file != "":
```
Comment (soimkim) [inline src/fosslight_binary/_jar_analysis.py:201]: Let's change this one.
` if json_file != "": -> if os.path.isfile(json_file ):`

File: src/fosslight_binary/_jar_analysis.py [hunk 3]
```diff
@@ -196,4 +197,11 @@ def analyze_jar_file(path_to_find_bin):
...
+            os.remove(json_file)
+    except Exception as ex:
+        logger.debug(f"Error - There is no .json file : {ex}")
+        success = False
```
Comment (soimkim) [inline src/fosslight_binary/_jar_analysis.py:205]: Remove `success = False`
Failing to delete temp files is a minor issue. Therefore, it does not affect success.

**Review by soimkim — CHANGES_REQUESTED** [review]

File: src/fosslight_binary/_jar_analysis.py (hunk 2 above)
Comment (bjk7119) [reply src/fosslight_binary/_jar_analysis.py:201]: got it

**Review by soimkim — APPROVED** [review]

**Review by soimkim — APPROVED** [review]

**Pull request merged**